### ISSUE TITLE
AWS_SES_REGION_ENDPOINT connection error

### ISSUE THREAD

**Issue by jihoon-MRX** [django-ses/django-ses#258]
If I assign ses smtp endpoint such as 'email-smtp.ap-northeast-2.amazonaws.com' to AWS_SES_REGION_ENPOINT on settings, it seems that it cannot be connected to the SES Server. 
However, if I assign region to AWS_SES_REGION_NAME only, it works perfectly.

Am I missing something or any updates for this?

**Comment by jihoon-MRX** [comment]
That tydo was in the message only, not in my project. Thanks tho, I corrected it.

**Comment by jihoon-MRX** [comment]
That tydo was in the message only, not in my project. Thanks tho, I corrected it.

**Comment by abheysharma23** [comment]
you found the solution?

**Comment by jmbreiner** [comment]
OH MY GOODNESS THANK YOU!!!!! This totally solved my problem. You can only define the AWS_SES_REGION_NAME or the AWS_SES_REGION_ENDPOINT. Not both. You are amazing.

**Comment by pcraciunoiu** [comment]
@jmbreiner would you open a PR to document this? Or do you recommend something else to help future people avoid this issue?

**Comment by ale180192** [comment]
In the documentation, there is an example of how to put the region name endpoint like `AWS_SES_REGION_ENDPOINT = 'email.us-west-2.amazonaws.com'`, and this value is passed directly to the boto client. However, in the boto client documentation, it says: `endpoint_url (string) – The complete URL to use for the constructed client. Normally, botocore will automatically construct the appropriate URL to use when communicating with a service. You can specify a complete URL (including the "http/https" scheme) to override this behavior. If this value is provided, then use_ssl is ignored.` This means we need to include the "https" scheme in the URL.

I believe we can update the documentation to avoid confusion with this requirement. Additionally, it's worth noting that the Django settings used by the SESBackend class is `AWS_SES_REGION_ENDPOINT_URL`, not `AWS_SES_REGION_ENDPOINT`. 

Here's the relevant code snippet from the `SESBackend` class:

super(SESBackend, self).init(fail_silently=fail_silently, **kwargs)
self._access_key_id = aws_access_key or settings.ACCESS_KEY
self._access_key = aws_secret_key or settings.SECRET_KEY
self._session_token = aws_session_token or settings.SESSION_TOKEN
self._region_name = aws_region_name if aws_region_name else settings.AWS_SES_REGION_NAME
self._endpoint_url = aws_region_endpoint if aws_region_endpoint else settings.AWS_SES_REGION_ENDPOINT_URL
self._throttle = cast_nonzero_to_float(aws_auto_throttle or settings.AWS_SES_AUTO_THROTTLE)
self._config = aws_config or settings.AWS_SES_CONFIG


I hope this clarification helps to improve the documentation and avoid potential issues with the SES endpoint URL.

**Comment by pcraciunoiu** [comment]
@ale180192 thanks for this detail. Would you be interested in opening a PR to clarify the documentation?

**Comment by ale180192** [comment]
Hi @pcraciunoiu, sure!! I'd like to send a PR. Do I have to create a fork or how can I send the PR?

**Comment by pcraciunoiu** [comment]
Yes @ale180192 you'd have to fork and use the GitHub editor or clone the repo and push to your fork

**Comment by MScatolin** [comment]
Based on this thread I was able to make it work using ACCESS_KEY and SECRET_KEY as variable names in settings.py (not the ones mentioned on readme. thank you @ale180192 

Also removed the endpoint var as suggested by @jihoon-MRX and @jmbreiner. Thanks!

+1 for the PR. At least on the readme, if nothing. I might give it a try later, if you like.

**Comment by pcraciunoiu** [comment]
@MScatolin you're welcome to PR it, I can review
